pkg/controller/catalog/storage/mem_storage.go

```diff
@@ -16,211 +16,45 @@ limitations under the License.
 
 package storage
 
-import (
-	"fmt"
-	"log"
-	"strings"
-
-	model "github.com/kubernetes-incubator/service-catalog/model/service_controller"
-)
-
-type bindingPair struct {
-	binding    *model.ServiceBinding
-	credential *model.Credential
-}
-
 type memStorage struct {
-	brokers map[string]*model.ServiceBroker
-	// This gets fetched when a SB is created (or possibly later when refetched).
-	// It's static for now to keep compatibility, seems like this could be more dynamic.
-	catalogs map[string]*model.Catalog
-	// maps instance ID to instance
-	services map[string]*model.ServiceInstance
-	// maps binding ID to binding
-	// TODO: support looking up all bindings for a service instance.
-	bindings map[string]*bindingPair
+	brokers        *memStorageBroker
```

Comment:
Or is "storage" the redundant part? If so, how would these types be named better? I'm open to suggestions, but `memBroker` doesn't sound right.

If anything, I think "mem" and "storage" should be reversed in the type names, leaving us with types like `memBrokerStorage` and `memInstanceStorage`. Thoughts?